public/plugin.ts

```diff
@@ -150,19 +151,21 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
       // logout the user when getting 401 unauthorized, e.g. when session timed out.
       core.http.intercept({
         responseError: (httpErrorResponse, controller) => {
```

Comment:
Please add a test to verify the new behavior

Reply:
Hi @peternied, thanks for the reply! I have tested it locally by viewing the session storage value after the `ttl` session got expired. Im working on adding some of the test to this PR now.